src/mmw/apps/modeling/geoprocessing.py

```diff
@@ -47,7 +48,27 @@
 }
 
 
-def histogram(polygons):
+def histogramPoll(jobId):
+    host = settings.GEOP['host']
+    port = settings.GEOP['port']
+    path = '/jobs/' + jobId
+    conn = httplib.HTTPConnection("%s:%s" % (host, port))
+
+    # Try for about two minutes ...
+    seconds = 1
+    for i in range(9):
```

Comment:
Is `9` just the number of loops it takes for the overall duration to reach ~2 minutes given the multiplier below?